package.json

```diff
@@ -41,21 +41,26 @@
   },
   "scripts": {
     "pre-commit": "yarn prettier:check && yarn lint",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "build:dev": "env-cmd -f ./.env.dev yarn build",
-    "dist": "env-cmd -f ./.env.prod yarn build",
-    "test": "react-scripts test",
-    "test:once": "cross-env CI=true env-cmd -f ./.env.test react-scripts test",
+    "start": "env-cmd -f ./.env.local react-app-rewired start",
+    "start:test": "env-cmd -f ./.env.test react-app-rewired start",
+    "start:ci": "react-app-rewired -r @cypress/instrument-cra start",
+    "build": "react-app-rewired build",
+    "dist": "env-cmd -f ./.env.production react-app-rewired build",
     "eject": "react-scripts eject",
     "hooks:uninstall": "husky uninstall",
     "hooks:install": "husky install",
     "lint": "eslint .",
+    "type-check": "tsc --noEmit",
     "prettier:check": "prettier --check 'src/**/*.js'",
     "prettier:write": "prettier --write 'src/**/*.js'",
-    "release": "git fetch --tags && git add CHANGELOG.md && standard-version -a",
-    "release:first": "git fetch --tags && git add CHANGELOG.md && standard-version -a --first-release",
-    "postinstall": "husky install"
+    "release": "standard-version -a",
+    "release:first": "standard-version -a --first-release",
+    "release:minor": "standard-version -a --release-as minor",
+    "release:major": "standard-version -a --release-as major",
+    "release:patch": "standard-version -a --release-as patch",
```

Comment:
Are we replacing those by release-please? @spaenleh

Reply:
@dialexo Yes, i think this will be the way to go, but not ready yet, it will require a configurable workflow in `.github` repo.